main.go

```diff
@@ -143,6 +174,15 @@ func JobInfoHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if name == "job-with-source-parameters" {
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		body := fmt.Sprintf(jobInfoWithSourceParameters, srv.Addr)
+		fmt.Println(body)
+		fmt.Fprint(w, body)
```

Comment:
Why do you have `fmt.Println(body)` just to run it through `Fprintf` right after?  Is it just to add a newline to the end of the body?  You don't really need it, right?

Reply:
`fmt.Println(body)` writes the body to stdout , `fmt.Fprint(w, body)` writes the body to the writer (http response)

Reply:
Oh, of course.  Sorry, I should have realized that.  :/